README.md

```diff
@@ -5,86 +5,115 @@
-* [🎯 Features](#-features)
-* [⚡️ Quickstart](#%EF%B8%8Fquickstart)
-* [📖 Documentation](#-documentation)
-* [💫 Project & community](#-project--community)
-* [📰 License](#-license)
+MicroCeph is an opinionated orchestration tool for Ceph clusters at all scales.
+It reduces the complexity of deploying and managing clusters by simplifying various operations like, service placement,
+disk administration and remote replication via RESTful APIs and CLI commands.
+
+Available as a snap, MicroCeph is the easiest tool for administrators, developers, and hobbyists to manage clusters.
+
+## Installation
+
+MicroCeph has first-class support as a snap. On snap-ready systems, you can install it on the command line with:
+
+```
+sudo snap install microceph
+```
+
+Disable automatic snap upgrades, so that no unexpected updates change your set-up:
+
+```
+sudo snap refresh --hold microceph
+```
+
+## Basic usage
+
+MicroCeph can deploy a Ceph cluster on a single machine with minimal commands.
+
+First, set up a Ceph cluster on your machine with:
+
+```
+sudo microceph cluster bootstrap
+```
+
+> [!NOTE]  
+> `cluster` is a MicroCeph subcommand for managing associated Ceph clusters.
+
+After setup, add storage to your cluster with:
+
+```
+sudo microceph disk add loop,4G,3
+```
+
+Here, you’ll add three virtual disks (“loop file” disks) of 4 GiB each.
+
+
+Once your cluster is set up and running, you can monitor its status with:
+
+```
+sudo microceph status
+```
+
+Note that there are no spaces between the `disk add` arguments.
 
-## 💡 Philosophy
 
-Deploying and operating a Ceph cluster is complex because Ceph is designed to
-be a general-purpose storage solution. This is a significant overhead for small
-Ceph clusters. MicroCeph solves this by being _opinionated_ and _focused_ on
-the small scale. With MicroCeph, deploying and operating a Ceph cluster is as
-easy as a [Snap][snap-microceph]!
+If you need a comprehensive status report of your cluster, including its health and disk usage, run:
 
-## 🎯 Features
+```
+sudo ceph status
+```
+> [!NOTE]
+> MicroCeph supports the usage of Ceph native tooling where snap-level commands are not yet available.
 
-* Quick and consistent deployment with minimal overhead
-* Single-command operations (for bootstrapping, adding OSDs, service enablement, etc)
-* Isolated from the host and upgrade-friendly
-* Built-in clustering so you don't have to worry about it!
+## Documentation
 
-## ⚡️ Quickstart
+The [MicroCeph documentation][rtd-microceph] contains guides and learning material about
+what you can do with MicroCeph and how it works.
 
-The below commands will set you up with a testing environment on a single
-machine using file-backed OSDs - you'll need about 15 GiB of available space on
-your root drive:
+Documentation is maintained in the [`docs`][docs-dir-microceph] directory of this repository.
+It is written in reStructuredTest (reST) format, built with Sphinx, and published on Read The Docs. 
```

Comment:
Suggest to make `Read The Docs` a link to RTD
(tiny nit: think they're also capitalized as `Read the Docs`)